### PR TITLE
Xposition issues fix #4, #9, and #10

### LINKED ISSUE #4: In adposition, add field for transliteration

### PR DESCRIPTION
**Pull request by ablodge**
adds transliteration field

### CODE REVIEW

**Comment by ablodge** [comment]
Please run
`python manage.py makemigrations`
`python manage.py migrate`
before running server

File: wiki/plugins/metadata/models.py [hunk 1]
```diff
@@ -555,11 +557,14 @@ def template(self):
...
         verbose_name = _('adposition')
 
+
 class AdpositionRevision(MetadataRevision):
 
     lang = models.ForeignKey(Language, related_name='adpositionrevisions', verbose_name='Language/dialect')
     # name = models.CharField(max_length=200, verbose_name='Lemma',
     #     help_text="Lowercase unless it would normally be capitalized in a dictionary")
+    transliteration = models.CharField(max_length=200, blank=True, verbose_name="Transliteration",
+        help_text="Pronunciation written in alphabetic letters")
```
[the remark this answers is not in the document]
Comment (ablodge) [reply wiki/plugins/metadata/models.py]: Sounds good.

**Comment by ablodge** [comment]
As for multiple transliterations, I'm not sure. Where this would probably make a difference is with Hebrew. Do you have an intuition of what would make the most sense there?

**Comment by nschneid** [comment]
Dunno. Doesn't seem crucial to standardize right now, so let's see how people use the transliteration field as is.

**Pull request merged**